install/kubernetes/cilium/values.yaml

```diff
@@ -515,3 +509,50 @@ global:
 
   cnpStatusUpdates:
     enabled: false
+
+# Configures the hubble-cli subchart
+hubble-cli:
+  # Enables the hubble-cli DaemonSet
+  enabled: false
+  # Configuration for the Hubble CLI
+  # image:
+    # repository of the docker image
+    # repository: quay.io/cilium/hubble
+    # tag is the container image tag to use
+    # tag: latest
+    # pullPolicy is the container image pull policy
+    # pullPolicy: Always
```

Comment:
just curious if we should uncomment PullPolicy (and tag as well), so that _make -C install/kubernetes all_ can update values based on version.

Reply:
These are resolved as defaults in the sub-chart where you should see them not commented, and thus taking effect. The parent chart is intended for overriding values, and the defaults from the sub-chart are provided as inline comments in the parent chart as a frame of reference.

Reply:
thanks, I was thinking that hubble-cli is part of this repo as well (i.e. release cycle is same with cilium), but it is maintained separately.